### ISSUE TITLE
Discussion: How critical is augeas?

### ISSUE THREAD

**Issue by patcon** [dwt/vagrant-hosts#6]
Firstly, this is a really awesome project, and I really appreciate your sharing it!

I've been using `vagrant-dns` for a bit (which leverages `rubydns`), but it seems there are too many moving parts and sometimes it just stops working. In retrospect, editing `hosts` seems to be the simplest means to make a vagrant project Just Work. It also seems to have a hope in heck of being cross-platform, and I don't think RubyDNS will ever work on Windows.

But it seems that the _augeas_ requirement is pretty high, especially if the goal is to create a plugin that makes setup a no-brainer even for beginners. There are some deb and rpm packages, but nothing for mac ppl save old macports or manual compilation. And augeas leaves windows users out in the cold, when other approaches for editing `hosts` might not be difficult to adapt to windows.

So in conclusion, how amenable might you be to considering hinging this project on an approach outside augeas? :)
